packages/expo-audio/src/AudioModule.web.ts

```diff
@@ -292,11 +297,20 @@ export class AudioRecorderWeb
     this._mediaRecorder?.pause();
   }
 
-  recordForDuration(seconds: number): void {}
+  recordForDuration(seconds: number): void {
+    this.record();
+    setTimeout(() => {
```

Comment:
Shouldn't we clear the timeout in some cases? Unhandled timeouts/intervals is never a good idea 😅